### PR TITLE
Update to Yocto Pyro

### PR DESCRIPTION
**Pull request by willnewton**

### CODE REVIEW

**Comment by willnewton** [comment]
This still requires a meta-resin release before merging.

**Comment by willnewton** [comment]
@agherzan @floion @telphan this PR should be ready to merge now.

File: layers/meta-resin-qemu/conf/samples/local.conf.sample [hunk 1]
```diff
@@ -66,3 +66,5 @@ PACKAGECONFIG_append_pn-qemu-native = " sdl"
 PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
 
 CONF_VERSION = "1"
+
+HOSTTOOLS += "docker iptables"
```
Comment (agherzan) [inline layers/meta-resin-qemu/conf/samples/local.conf.sample]: Why iptables?

File: qemux86-64.coffee [hunk 2]
```diff
@@ -38,7 +38,7 @@ module.exports =
 		machine: 'qemux86-64'
```
Comment (agherzan) [inline qemux86-64.coffee]: NIT: maintain the same commit format.

File: layers/meta-resin-qemu/conf/samples/local.conf.sample (hunk 1 above)
Comment (willnewton) [reply layers/meta-resin-qemu/conf/samples/local.conf.sample]: iptables is a runtime dependency of dockerd

**Pull request merged**